de_workloads/data_processing/silver/spark_jobs/silver.py

```diff
@@ -1,10 +1,42 @@
-from pysparkle.pysparkle_cli import cli
+import logging
 
+from pysparkle.etl.etl import get_spark_session_for_adls, save_files_as_delta_tables
+from pysparkle.logger import setup_logger
+from pysparkle.storage_utils import get_adls_directory_contents
+from pysparkle.utils import filter_files_by_extension
 
-def call_pysparkle_entrypoint():
-    """Execute silver processing in Pysparkle."""
-    cli(["silver", "--dataset-name", "movies_dataset"], standalone_mode=False)
+
+BRONZE_CONTAINER = "raw"
+SILVER_CONTAINER = "staging"
+
+logger_library = "pysparkle"
+logger = logging.getLogger(logger_library)
+
+
+def silver_main(dataset_name: str) -> None:
```

Comment:
As above, perhaps `data_processing_main` or `etl_main` rather than silver? Fine for the example workload to be called silver, but the function that's generated will be from a jinja template ultimately.

Reply:
done